instant-xml-macros/src/ser.rs

```diff
@@ -67,6 +67,10 @@ fn serialize_struct(
     data: &syn::DataStruct,
     meta: ContainerMeta,
 ) -> proc_macro2::TokenStream {
+    let ident = &input.ident;
+    let tag = meta.tag();
```

Comment:
Would prefer to bind `tag` and `default_namespace` as close to their usage site as makes sense (similar to where they were before) -- same for all the users.

Why do we need to take a reference of `meta.default_namespace()`?